tests/indico_wrapper/test_modelops.py

```diff
@@ -0,0 +1,33 @@
+"""
+Test Model Ops class methods
+"""
+import pytest
+import json
+from indico_toolkit.indico_wrapper.modelop import ModelOp
+
+
+@pytest.fixture(scope="module")
+def modelop(indico_client):
+    return ModelOp(indico_client)
+
+
+@pytest.fixture(scope="module")
+def model_setting(extraction_model_group_id, modelop):
+    model = modelop.get_model_group(extraction_model_group_id)
+    setting = json.loads(model["modelOptions"]["modelTrainingOptions"])
+    if setting is None or "auto_negative_scaling" not in setting:
+        updated_model = modelop.update_model_settings(
```

Comment:
you don't need to call update_model_settings here-- it's none can't you just return "True"? (or False) i.e. it should be fine to switch it either way and validate it

Reply:
(importantly, you're testing this method in particular so shouldn't really use it to setup the validation for your test)